sync/syncer.go

```diff
@@ -288,11 +289,16 @@ func (s *Syncer) BlockSyntacticValidation(block *types.Block) ([]*types.Addressa
 		return nil, nil, errors.New(fmt.Sprintf("data availabilty failed for block %v", block.ID()))
 	}
 
-	//validate blocks view
+	//validate block's view
 	if valid := s.ValidateView(block); valid == false {
 		return nil, nil, errors.New(fmt.Sprintf("block %v not syntacticly valid", block.ID()))
 	}
 
+	//validate block's votes
```

Comment:
i'v seen you also integrated this to the validation_queue in that case you can put this in the callback we create for the original block